FE/frontend/src/util/config.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  BASE_ENDPOINT_URL: 'https://localhost:7151',
```

Comment:
Let's move this to environment variable?

We should not push API endpoints to Git repo. Instead, make an env variable called `REACT_APP_BASE_ENDPOINT_URL` and then use that here?